types/yup/yup-tests.ts

```diff
@@ -1152,6 +1152,26 @@ yup.object({
         }
         return true;
     }),
+    alias: yup
+        .string()
+        .defined()
+        .test({
+            name: 'test1',
+            test: (value: string | null | undefined) => true,
```

Comment:
Instead of putting a type annotation on these, can you leave it unannotated and then use an `$ExpectType`?